packages/core/src/test/techdebt.test.ts

```diff
@@ -42,4 +44,18 @@ describe('tech debt', function () {
             'with node16+, we can use crypto.randomUUID and remove the "uuid" dependency'
         )
     })
+
+    it('feature/standalone branch temporary debug log level for testing', async function () {
+        assert.strictEqual(
+            defaultLogLevel,
+            'info',
+            'set loglevel defaults back to info for src/shared/logger/activation.ts. (revert this commit)'
+        )
```

Comment:
Good idea. To avoid failures on feature/standalone, we could skip this check if [GITHUB_HEAD_REF](https://docs.github.com/en/actions/learn-github-actions/variables) contains "standalone"